src/lib/components/Switch.svelte

```diff
@@ -3,7 +3,13 @@
 	export let name: string;
 </script>
 
-<input bind:checked type="checkbox" {name} class="peer pointer-events-none absolute opacity-0" />
+<input
+	bind:checked
+	type="checkbox"
+	{name}
+	id={name}
```

Comment:
The id added here breaks the functionality of the switch for me. If it was added for accessibility, could we use `aria-label` instead? @gary149